ui/pages/import-tokens-confirmation/index.scss

```diff
@@ -0,0 +1,8 @@
+.tokens-container {
+  display: flex;
+  flex-direction: column;
+  overflow: auto;
+  max-height: 550px;
+  margin-left: 16px;
+  margin-right: 16px;
+}
```

Comment:
Since we have replaced tokens-container with Box, can we handle this via box Props. This way we can eliminate this file